components/dashboard/cards/testimonial.tsx

```diff
@@ -0,0 +1,67 @@
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from '@/lib/redux/store'
+import { setTestimonials, setDisplay } from '@/lib/redux/slices/testimonials'
+import { StyledTestimonialCard, StyledTestimonialInput } from './StyledCards'
+import { Close } from '@mui/icons-material'
+import { Button } from '@mui/material'
+import { useState } from 'react'
+import { createTestimonials } from '@/lib/db/cms/testimonials'
+
+const TestimonialCard = ({ field }: any) => {
+    const dispatch = useDispatch<AppDispatch>()
+    const [isFlipped, setIsFlipped] = useState(false)
+    const { display, email, id, name, rating, review } = field
+
+    const handleDisplayChange = (event: {
+        target: { id: string; checked: boolean }
+    }) => {
+        const { id, checked } = event.target
+        dispatch(setDisplay({ id, checked }))
+    }
+
+    const toggleDeletionView = event => {
+        setIsFlipped(!isFlipped)
+    }
+    return (
+        <StyledTestimonialCard>
+            <div className={isFlipped ? 'card is-flipped' : 'card'} id={id}>
```

Comment:
className={`card ${isFlipped ? 'isFlipped' : ''`}. Also, let's make the class names camel cased. Too much flip flopping, all variables and classes camel cased no word-word-word casing